client/src/components/CrossCheck/criteria/SubtaskCriteria.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Rate, Typography } from 'antd';
+import { Input } from 'antd';
+
+import { CrossCheckCriteriaData, ICommentState, ICountState } from '../CrossCheckCriteriaForm';
+
+interface SubtaskCriteriaProps {
+  criteriaComment: ICommentState[];
+  task: CrossCheckCriteriaData;
+  countStar: ICountState[];
+  updateCountStar: (event: number, max: number, key: string) => void;
+  updateComment: (value: string, key: string) => void;
+}
+
+export default function SubtaskCriteria({
+  criteriaComment,
+  task,
+  countStar,
+  updateCountStar,
+  updateComment,
+}: SubtaskCriteriaProps) {
+  const { TextArea } = Input;
+
+  const criteriaScore = task ? (countStar.find(item => item.key === task.key)?.point as number) : 0;
+
+  return (
```

Comment:
I've refactored this line, but I haven't changed it to null or undefined, because I need to have this variable as number in any case. If this criteria wasn't checked before the score for it should be equal to 0.